pkg/api/applications/v2/http.go

```diff
@@ -39,6 +39,23 @@ type httpAPI struct {
 
 var _ API = &httpAPI{}
 
+func (h *httpAPI) CheckEndpoint(ctx context.Context) (api.Metadata, error) {
+	req, err := http.NewRequest(http.MethodHead, h.client.URL(endpointApplications).String(), nil)
```

Comment:
I think I just merge conflicted you by reading my email in chronological order. This ends up being `h.endpoint` instead of `endpointApplications`.